app/controlplane/cmd/wire.go

```diff
@@ -96,3 +99,16 @@ func newCASServerOptions(in *conf.Bootstrap_CASServer) *biz.CASServerDefaultOpts
 		DefaultEntryMaxSize: in.GetDefaultEntryMaxSize(),
 	}
 }
+
+func newSigningCAs(cas []*conf.CA, logger log.Logger) (*ca.CertificateAuthorities, error) {
```

Comment:
nice, no need to do it explictly

Reply:
Yes, it was not using wire, not sure why I didn't do it at that time.